web3.js/src/message.ts

```diff
@@ -129,7 +129,7 @@ export class Message {
       return {
         programIdIndex,
         keyIndicesCount: Buffer.from(keyIndicesCount),
-        keyIndices: Buffer.from(accounts),
```

Comment:
This is actually a `seq()` of `u8` in the layout schema, ergo I believe this decodes as `number[]`.

Reply:
Ah, I see. Well, no problem then, as long as it doesn't change the API I think it's good.